assets/src/editor/events/tickets/editor-ticket/edit-form/use-ticket-form-loading-state.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External imports
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * @return {Object} object for tracking ticket form loading state
+ * @member {boolean} loadingTicketForm
+ * @member {Function} setLoadingTicketForm
+ */
+const useTicketFormLoadingState = () => {
+	const [ loading, setLoading ] = useState( false );
+	return {
+		loadingTicketForm: loading,
+		setLoadingTicketForm: setLoading,
+	};
+};
+
+export default useTicketFormLoadingState;
```

Comment:
Why is this specific to Ticket form? It can have a generic use. Also, does it it do anything more than `useState()`?